lib/hanami/environment.rb

```diff
@@ -420,17 +408,8 @@ def code_reloading?
 
     # @since 0.4.0
     # @api private
-    def architecture
-      @options.fetch(:architecture) do
-        puts "Cannot recognize Hanami architecture, please check `.hanamirc'"
-        exit 1
-      end
-    end
-
-    # @since 0.4.0
-    # @api private
     def container?
-      architecture == CONTAINER
+      true
```

Comment:
This is the smelliest part of the code. We only support CONTAINER, so I just made it return true.

I could go further and remove all references to `.container?` but there are quite a few and it involves several different sections of the codebase.